lib/pages/new_private_chat/create_room_controller.dart

```diff
@@ -0,0 +1,37 @@
+import 'package:fluffychat/app_state/success.dart';
+import 'package:fluffychat/di/global/get_it_initializer.dart';
+import 'package:fluffychat/domain/app_state/room/create_room_success.dart';
+import 'package:fluffychat/domain/model/room/new_room_request.dart';
+import 'package:fluffychat/domain/usecase/create_room_interactor.dart';
+import 'package:flutter/widgets.dart';
+import 'package:matrix/matrix.dart';
+import 'package:vrouter/vrouter.dart';
+
+class CreateRoomController {
+  final _createRoomInteractor = getIt.get<CreateRoomInteractor>();
+
+  void createRoom(BuildContext context, Client matrixClient, NewRoomRequest newRoomInformations) {
+    _createRoomInteractor.execute(matrixClient, newRoomInformations).listen(

```

Comment:
- Create StreamSubscription? streamSubcription;
- Assign it
```suggestion
 streamSubcription = _createRoomInteractor.execute(matrixClient, newRoomInformations).listen(
```
- Cancel it inside `dispose()` of StatefulWidget